README.md

```diff
@@ -1,3 +1,201 @@
+* include an approximate distance calculator as well as a travel cost estimator, so that employers can be aware of the distances that the job seekers have to travel,
+* And take this in consideration when calculating remuneration package.
+* Information could also be used so that job seekers who live close together can pool resources to arrange transport to the weekly sessions, and thus save costs.
+
+To-do list:
+* Wow improvement to be done 1
+* Wow improvement to be done 2
+
+## Status
+Project is: _in progress_, and why?
+
+## Inspiration
+Thank you to Andela and Facebook for giving us the opportunity to work on this project.
+, our team mentor - thank you for all your patience and advice!
+Did you use someone else’s code?
+Do you want to thank someone explicitly? Did someone’s blog post spark off a wonderful 
+idea or give you a solution to nagging problem? The labour bureau at Lynnwoodrif Dutch 
+Reformed Church in Pretoria Gauteng sparked the idea for the project.
+One of our team members also apply for tutoring jobs through a tutoring agency that uses 
+TutorCruncher, so this may have impacted the idea for a potential solution.
+
+## Contributing
+If this project sounds interesting to you and you'd like to contribute, thank you! First, 
+you can send a mail to [@buildforsdg@andela.com](https://www.buildforsdg@andela.com) to indicate your interest, why you'd like 
+to support and what forms of support you can bring to the table, but here are areas we 
+think we'd need the most help in this project :
+
+1.	Area one - this app is deals with human trafficking and you need feedback on
+    your roadmap and feature list from the private sector / NGOs
+2.	Area two - you want people to opt-in and try using your staging app at 
+    staging.project-name.com and report any bugs via a form
+3.	Area three - here is the zoom link to our end-of sprint webinar, join and 
+    provide feedback as a stakeholder if you can
+	
+## Authors
+
+BuildforSDG Challenge 2020 - Team 258 - Fumana Project- Team Members
+
+|Picture|Team Role|Name|Country|City|Github|
+|---|---|---|---|---|---|
+|![site map detailed](./fumanaImg/fumana_logo_1.png)|Team Assistant|Emilia|Country|City|Github|
+|![site map detailed](./fumanaImg/fumana_logo_1.png)|Team Mentor|Name|Country|City|Github|
+|![site map detailed](./fumanaImg/relax.jpg)|Team Leader|Dibwe Kalangu|South Africa|Johannesburg|leanny01|
+|![site map detailed](./fumanaImg/fumana_logo_1.png)|Project Manager|Sphesihle Zulu |South Africa|KwaZulu Natal|sphezulu|
+|![site map detailed](./fumanaImg/JPEG_20200814_081646.jpg)|Front-End Dev|Emmanuel Uchenna|Nigeria |Ebony State|eunit99|
+|![site map detailed](./fumanaImg/kelv_mutiso.png)|UI/UX Dev|Kelvin Mutiso|Kenya |Nairobi|mutisikelv|
+|![site map detailed](./fumanaImg/IMG_9603_comp.jpg)|Back-End Dev|Soninake Oluwadamilola |Nigeria |Lagos |damiso15|
+|![site map detailed](./fumanaImg/fumana_logo_1.png)|Front-End Dev|Chaibo Seopa|South Africa |Cape Town|Github|
```

Comment:
1.  Please add `Sphesihle Zulu`
2. The following member is to be removed as she hasn't reported to the team since the last time she was introduced
`Chaibo Seopa`

Reply:
sphesihle is on line 188